docs-archived/guides/dsm_azure_sobject.md

```diff
@@ -237,11 +194,10 @@ resource "dsm_azure_sobject" "rsa_key_azure" {
   soft_deletion = true
 }
 
-
 ## Purging a dsm_azure_sobject.
 
 # Enable purge_deleted_key as true.
-# This can be enabled only during update.
+# This can be enabled only after soft_deletion can this be enabled during an update
```

Comment:
The suggestion here @ravigfortanix , was to replace the line with the one above because the current revision muddies things a bit more.
- Current:  `# This can be enabled only after soft_deletion can this be enabled during an update`
- Suggested: `# "Only after soft_deletion can this be enabled during an update"`

Reply:
My bad. I overlooked into it. Thanks for correcting it.